class CompanyNotFoundException(Exception):
    pass


class CompanyAlreadyExistsException(Exception):
    pass
